### PR TITLE
✨ Added. Artifacts Class

### PR DESCRIPTION
**Pull request by KristoferSoler**

### CODE REVIEW

**Comment by KristoferSoler** [comment]
All yours @pyasi ^^ Take a look now.

**Comment by KristoferSoler** [comment]
@pyasi now I see that Emojis doesn't return a Client either as the Annotation. Do you want me to remove that ":return: ...." ?

File: pybuildkite/buildkite.py [hunk 1]
```diff
@@ -102,17 +103,25 @@ def agents(self):
...
         Get Annotation operations for the Buildkite API
         """
         return Annotations(self.client, self.base_url)
 
+    @requires_token
+    def artifacts(self):
+        """
+        Get Artifact operations for the Buildkite API
+
+        :return: Client
```
Comment (pyasi) [inline pybuildkite/buildkite.py]: remove this return

File: pybuildkite/artifacts.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        :param pipeline: pipeline slug
+        :param build: build number
+        :param job: job id
+        :return: Returns a paginated list of a build’s artifacts across all of its jobs.
+        """
+        url = self.path + "jobs/{}/artifacts/"
+        return self.client.get(url.format(organization, pipeline, build, job))
+
+    def get_artifact(self, organization, pipeline, build, job, artifact):
+        """
```
Comment (pyasi) [inline pybuildkite/artifacts.py]: Missing documentation of the method

File: pybuildkite/artifacts.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        :param build: build number
+        :param job: job id
+        :param artifact: artifact id
+        :return:
+        """
+        url = self.path + "jobs/{}/artifacts/{}/"
+        return self.client.get(url.format(organization, pipeline, build, job, artifact))
+
+    def download_artifact(self, organization, pipeline, build, job, artifact):
+        """
```
Comment (pyasi) [inline pybuildkite/artifacts.py]: Missing documentation of the method

File: pybuildkite/artifacts.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+    def get_artifact(self, organization, pipeline, build, job, artifact):
+        """
+
+        :param organization: organization slug
+        :param pipeline: pipeline slug
+        :param build: build number
+        :param job: job id
+        :param artifact: artifact id
+        :return:
```
Comment (pyasi) [inline pybuildkite/artifacts.py]: Missing return here and on the one below as well

File: pybuildkite/artifacts.py (hunk 2 above)
Comment (KristoferSoler) [reply pybuildkite/artifacts.py]: There is no info on the page https://buildkite.com/docs/apis/rest-api/artifacts#get-an-artifact 
But I put "Return an artifact". ¿It's ok?

File: pybuildkite/artifacts.py (hunk 3 above)
Comment (KristoferSoler) [reply pybuildkite/artifacts.py]: Done.

File: pybuildkite/artifacts.py (hunk 4 above)
Comment (KristoferSoler) [reply pybuildkite/artifacts.py]: Done.

File: pybuildkite/buildkite.py (hunk 1 above)
Comment (KristoferSoler) [reply pybuildkite/buildkite.py]: Done.

**Pull request merged**